src/tabs/tab-header-group.component.ts

```diff
@@ -191,18 +242,12 @@ export class TabHeaderGroup implements AfterContentInit, OnDestroy, OnChanges {
 		if ((event.key === " " || event.key === "Spacebar") && !this.followFocus) {
 			tabHeadersArray[this.currentSelectedIndex].selectTab();
 		}
-
-		// dropdown list handler
-		if (event.key === "Escape") {
-			this.tabListVisible = false;
-		}
 	}
 
-	@HostListener("focusout", ["$event"])
-	focusOut(event) {
-		if (this.tabListVisible && !this.elementRef.nativeElement.contains(event.relatedTarget)) {
-			this.tabListVisible = false;
-		}
+	ngOnInit() {
+		this.windowResizeSubscription = fromEvent(window, "resize", { passive: true })
+			.pipe(debounceTime(200))
+			.subscribe(() => this.handleScroll());
```

Comment:
Would the `DocumentService` or `EventService` in `utils/UtilsModule` help here? Both of them automatically handle and clean up subscriptions, which would make this a bit cleaner

Reply:
No debounce, but honestly all that usually does is slow down the UI - most changes are fast enough without a debounce that it feels way better.

Reply:
The debounce makes it function equivalently to reference implementation.

Reply:
Do we need it though? Playing with the react implementation, it definitely makes it a bit weird (the overflow button hangs around for a while after it should disappear), and 200ms is pretty short - how much does that really improve performance?

Either way, if we do need it we can adjust our services as needed, and my original question is still relevant: can we use `DocumentService` or `EventService` for the resize events? Auto-cleanup is a pretty compelling bonus I think.

Reply:
From a usability perspective, `EventService` takes gold compared to `fromEvent`